webapp/channels/src/components/channel_header_mobile/index.ts

```diff
@@ -6,7 +6,7 @@ import {connect} from 'react-redux';
 import {bindActionCreators, Dispatch} from 'redux';
 import {withRouter, matchPath} from 'react-router-dom';
 
-import {createSelector} from 'reselect';
+import {createSelector} from 'packages/reselect/src';
```

Comment:
@hmhealey any suggestion or changes. I have also considered at one point to just change it to `mattermost-reselect` but 0/5

Reply:
Maybe move `packages/reselect/src/index` to `packages/mattermost-redux/src/selectors/create_selector`? That way, it'll be accessible all throughout the web app, and we can remove all of the reselect-specific import rewriting from various configuration files

Reply:
yup that looks good too.

Reply:
Thanks!